src/_help/enhance-documentation-content/topics.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Topics
+---
+
+- TOC
+{:toc}
+
+Topics are an exclusive custom property that we've created to allow you to add extended content to your API documentation. They enable you to add context, additional information, or even tutorial elements to guide your readers in using your API.
+
+By default, most specification standards do not support the addition of extra content. So we have added an additional property to insert at the beginning of your specification file: `x-topics`.
```

Comment:
I think we can remove this sentence following my previous proposal, as we directly mention the property. I find it more clear like this, WDYT?